webapp/src/main/java/org/apache/atlas/web/rest/EntityREST.java

```diff
@@ -1952,7 +1954,47 @@ public void repairAccessControlAlias(@PathParam("guid") String guid) throws Atla
        } finally {
               AtlasPerfTracer.log(perf);
        }
+    }
+
```

Comment:
Can we move this to a separate class? `EntityREST` class is way too big. Perhaps `BusinessPolicyCRUD` as the class name?